app/index.js

```diff
@@ -1,17 +1,27 @@
 import { setup } from './insights.js'
-import 'log-timestamp'
-import { createServer } from './server.js'
+import { startServer } from './server.js'
 
 const init = async () => {
-  const server = await createServer()
+  const server = await startServer()
```

Comment:
a nitpic I know but `startServer()` doesn't the right name given the next line is `server.start()` 🤔

Reply:
good point!